src/test/java/com/wikia/webdriver/testcases/oasismainpage/CuratedContentModal.java

```diff
@@ -18,4 +20,14 @@ public void CuratedContentModal_001_modalVisible() {
     CuratedContentToolModal modal = main.clickCuratedContentToolButton();
     Assertion.assertTrue(modal.isModalVisible());
   }
+
+  @Execute(asUser = User.STAFF)
+  @Test(groups = {"CuratedContentModal_002", "CuratedContent"})
+  public void CuratedContentModal_002_MercuryNavBarShouldNotBeVisible() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)